src/components/region-common.js

```diff
@@ -11,7 +11,9 @@ import * as Utils from 'src/libs/utils'
 // https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/reference.conf
 export const unknownRegionFlag = '❓'
 export const getRegionInfo = (location, locationType) => {
-  const regionDescription = `${locationType}: ${location}`.toLowerCase()
+  const regionDescription = locationType === locationTypes.multiRegion ?
+    `${location} (${locationTypes.multiRegion})` :
+    `${location}`.toLowerCase()
```

Comment:
With the simpler string, interpolation is no longer needed
```suggestion
    location.toLowerCase()
```

Reply:
Thanks! What this code ended up doing here was checking for `location`. If `location` is undefined, `location.toLowerCase()` throws an error, while `` `${location}`.toLowerCase() `` does not. 

I've fixed this to not use interpolation and also check for `location`.